### ISSUE TITLE
When Italian language is applied in the app, in the projects dialog "Add project” and "About” aren’t aligned properly.

### ISSUE THREAD

**Issue by dbemke** [getodk/collect#5554]
#### ODK Collect version
the store version 2023.1.2, the master version 1f9155c7f1f6c1d91532993fd45cc9b68270124a
#### Android version
10, 13
#### Device used
Redmi 9T, Galaxy A23
#### Problem description
When Italian language is applied in the app, in the projects dialog "Add project” and "About” aren’t aligned properly.

![italianosettingsdialog](https://user-images.githubusercontent.com/56479916/233045368-a73ec91b-b3fd-4f5e-af78-5e31cef237d3.png)

#### Steps to reproduce the problem
1. Go to Settings→ User Interface→ Language and choose "italiano".
2. Tap the project icon to open the projects dialog.
3. Check "Add project” and "About” in the dialog.

**Comment by seadowg** [comment]
Is this still a problem in #5732?

**Comment by grzesiek2010** [comment]
> Is this still a problem in https://github.com/getodk/collect/issues/5732?

Yes because we eventually didn't change the way those buttons are displayed. As I said in https://github.com/getodk/collect/pull/5770#issuecomment-1772454670 I think It would be good to convert that dialog to a full-screen one to fix those problems with lack of space.

**Comment by seadowg** [comment]
I'm thinking it might be possible to stack the buttons vertically if we run out of room with the current design. Would it be possible to have a quick look at that as part of #5770 @grzesiek2010?

**Comment by grzesiek2010** [comment]
> I'm thinking it might be possible to stack the buttons vertically if we run out of room with the current design. Would it be possible to have a quick look at that as part of

I already gave it a try. It was one of the versions of that pr before merging the final one. The problem was that then we didn't have enough space for the list of projects so it seems like there is no perfect solution. Even if we fix one thing it impacts something else.
As I said the only solution seems to be using a full-screen dialog in this case what @alyblenkin will consider.

**Comment by alyblenkin** [comment]
@grzesiek2010 @seadowg - agreed, I think a full-screen dialog will accommodate the design changes we wanted to make in the previous pr. My assumption was we would prioritize this for the next release. What do you think @lognaturel?

**Comment by lognaturel** [comment]
I do like the dialog here because it makes it feel to me like projects are a concept that's layered on. That said, I don't have strong feelings other than we should make sure not to spend too much time on it because it hasn't been a significant pain point!

**Comment by alyblenkin** [comment]
Okay, so we can leave it for now and maybe come back to it later. I'll note it down in my design tracker.

**Comment by lognaturel** [comment]
Another quick thing I remembered that should be captured somewhere -- we picked a dialog because in general there won't be very much content. If we go full screen, it will generally have a lot of blank space in it.

**Comment by alyblenkin** [comment]
> Another quick thing I remembered that should be captured somewhere -- we picked a dialog because in general there won't be very much content. If we go full screen, it will generally have a lot of blank space in it.

Yes, I'm glad you captured this. If we stick with the dialog and we get user feedback about this being a pain point, I think we could explore stacking the buttons vertically again. We would be optimizing for fewer projects, but maybe that's OK if we want to address this issue.
